deploy/kubedirector/kubedirector.hpe.com_kubedirectorapps_crd.yaml

```diff
@@ -127,6 +129,8 @@ spec:
                       packageURL:
                         type: string
                         pattern: '^(file|https?)://.+\.tgz$'
+                      useNewSetupLayout:
+                        type: boolean
```

Comment:
Should we also update our example CRs to set this field

Reply:
Yep good idea.

Reply:
The annoying thing of course is that any of them may need to get their persistDirs corrected to work in the new layout. I'll see which ones are obvious easy changes first.

Reply:
Oh yeah. 
We can just pick one, centos?